src/app/ViewController.ts

```diff
@@ -85,11 +85,23 @@ export class ViewController {
     if (this.tradeModal) {
       this.tradeModal.setNextPlayer(player);
     } else {
-      this.tradeModal = new TradeModal(trade, player, this);
+      this.tradeModal = new TradeModal(
+        trade,
+        player,
+        () => this.runTimer(),
```

Comment:
Is naming these callback something that would make the code more readable later  ?